specs/agents/tracing-instrumentation-db.md

```diff
@@ -44,13 +44,13 @@ The following fields are relevant for database and datastore spans. Where possib
 
 | Field | Value / Examples | Comments |
 |-------|:---------------:|----------|
-|`name`| e.g. `DynamoDB UpdateItem my_table`|  The span name should capture the operation name (as used by AWS for the action name) and the table name, if available. The format should be `DynamoDB <ActionName> <TableName>`. |
+|`name`| e.g. `DynamoDB UpdateItem my_table`|  The span name should capture the operation name (as used by AWS for the action name) and the table name, if available. The format should be `DynamoDB <ActionName> <TableName>`. TableName MAY be omitted from the name for operations (`batchWriteItem`, `batchGetItem`, PartiQL-related methods like `executeStatement` etc.) that are acting on more than a single table. If `TableName` is not available, agents SHOULD also check the `TableArn` or `SourceTableArn` query params for a table name.|
```

Comment:
1. Do `TableArn` and `SourceTableArn` have the same (magnitude) of cardinality as the table names.
   Or can there be cases that the `*Arn` will have a high cardinality? 
   We always need to keep in mind that, in general, we need to limit the cardinality of span names.  
   If that's not an issue here, then I'm fine with the above proposal!
2. Maybe we should also explicitly state here that PartiQL-related methods **must not** contain the query in the name (again: cardinality).

Reply:
> Do TableArn and SourceTableArn have the same (magnitude) of cardinality as the table names.

An example TableArn is `arn:aws:dynamodb:us-west-2:123456789012:table/$tableName`. Perhaps we could specify that the `$tableName` should be *extracted* from the ARN, rather than using the full ARN?

> PartiQL

There is some similarity in choosing a span.name for a PartiQL expression with a span.name for GraphQL: a single expression that can operate on multiple tables/classes. ([This comment](https://github.com/elastic/apm/issues/446#issuecomment-858137915) summarizes what the Node.js agent does for GraphQL span names.) However, I'm not sure we want to get into that yet here. IOW, I agree with "MAY be omitted ... for ... PartiQL".

Reply:
Thanks guys -- Re: PartiQL I've updated the text to encourage folks _not_ to include this data in order to reduce the cardinality of captured data.

Re: `TableArn` and `SourceTableArn` -- for most users I don't believe capturing a table's ARN instead of it's name introduces a higher cardinality of captured values.  DynamoDB offers a single table space.  However, capturing the entire ARN does seem excessive here so I've updated the text to indicate that table name should be extracted.